src/components/Visualization/useAnalyticsData.js

```diff
@@ -80,7 +81,15 @@ const fetchAnalyticsData = async ({
         }
     }
 
-    const rawResponse = await analyticsEngine.events.getQuery(req)
+    const analyticsApiEndpoint =
+        visualization.outputType === OUTPUT_TYPE_ENROLLMENT
```

Comment:
There will be more endpoints later, so maybe go with a mapping object right away.

Reply:
Maybe analyics endpoint could be added to the `outputTypeMap` in https://github.com/dhis2/event-reports-app/blob/develop/src/modules/visualization.js#L29?

Reply:
Ok, I guess having this mapping in `useAnalyticsData` is fine since I don't expect it to be needed in other places.

Reply:
Ok